src/page.es6

```diff
@@ -1,15 +1,13 @@
-;(function(root) {
-  var document = root.document,
-      VARIANTS_DELIMITER = '+';
-
-  // Internal: Initializes a 'Page' object.
-  // Browser environments does not need to call this constructor
-  // since a 'page' object will be exported in the global namespace.
-  var Page = function() {
-    this.initializers = {};
-    this.initializers[':before'] = [];
-    this.initializers[':after'] = [];
-  };
+// Internal: Initializes a 'Page' object.
+// Browser environments does not need to call this constructor
+// since a 'page' object will be exported in the global namespace.
+class Page {
+  static variantsDelimiter() { return '+'; }
```

Comment:
I wanted to mimic a class level constant here (previously this was a private variable) but even which the `static` here this still can be mutated by the outside world.